src/pages/docs/layouts.mdx

```diff
@@ -49,12 +55,13 @@ horizontal fashion. Each child occupies the same amount of space, i.e. for n chi
 }
 ```
 
-<DemoButton to='/examples/layouts#horizontallayout' label="Demo"/>
+<DemoButton to='/examples/layouts#horizontal-layout' label='Demo' />
```

Comment:
AFAIK the anchors are generated, so is this working with the additional hyphen?

Reply:
It does not work with the hyphen and it does not work with it either. I searched and this is a bug in docz. I removed the hyphen.